plugins/techdocs/package.json

```diff
@@ -132,6 +132,41 @@
                     }
                   }
                 }
+              },
+              {
+                "type": "object",
```

Comment:
It might be worth moving this out into a `schema.d.ts` file and using the `files` entry instead here.

Reply:
Since https://github.com/backstage/backstage/pull/3799 is doing this for all techdocs packages, I think it's okay to skip it here.